index.html

```diff
@@ -1752,7 +1752,7 @@ <h2>
         [Exposed=Window]
 
         interface GamepadEvent: Event {
-          constructor(DOMString type, GamepadEventInit eventInitDict);
+          constructor(DOMString type, optional GamepadEventInit eventInitDict = {});
```

Comment:
We probably need to remove the `required` from GamepadEventInit's `gamepad`.

Reply:
Blink, Gecko, WebKit agree that GamepadEventInit's `gamepad` attribute is optional and nullable, let's update the spec to match.

```
  Gamepad? gamepad;  // blink
  Gamepad? gamepad = null;  // Gecko
  Gamepad? gamepad = null;  // WebKit
```